src/chroot_script

```diff
@@ -212,6 +212,25 @@ echo "mapping wlan1" >> /etc/network/interfaces
 echo "  script /root/bin/map_iface" >> /etc/network/interfaces
 echo "source /boot/octopi-network.txt" >> /etc/network/interfaces
 
+# allow wifi network configuration via /boot/octopi-wifi.txt
+mv /etc/wpa_supplicant/wpa_supplicant.conf /boot/octopi-wifi.txt
+ln -s /boot/octopi-wifi.txt /etc/wpa_supplicant/wpa_supplicant.conf
```

Comment:
You are right, might be overkill. I'm just worried that people get confused. Then again, there are so many other files, and they don't touch those either, so that should hopefully work out.

This whole discussion tickled my memory though and I remembered this:

>  If a wpa_supplicant.conf file is placed into the /boot/ directory, this will be moved to the /etc/wpa_supplicant/ 
> directory the next time the system is booted, overwriting the network settings; this allows a Wifi configuration to 
> be preloaded onto a card from a Windows or other machine that can only see the boot partition.

That's from the [blog post](https://www.raspberrypi.org/blog/another-update-raspbian/) about the release of the most recent Raspbian image. My guess would be that this is also in place on the lite image (which is used for building OctoPi, although the current OctoPi 0.13 build is still based on one raspbian version before what the above post is about - perfect timing once again, they released a new raspbian version merely a week after we had gone through our whole RC pretesting and release cycle...) and hence should already work. Have you tested or can you test this? It wouldn't make much sense to add some other means to achieve the same goal if this already worked out of the box this way I guess :)